main.ts

```diff
@@ -9,7 +9,7 @@ import { globals } from './main-globals';
 const codeRunningOnMac: boolean = process.platform === 'darwin';
 
 // ========================================================================================
-// *FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER*** BUILD TOGGLE *FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER**FOLDER*
```

Comment:
lololol

Reply:
I recommend not viewing commits one by one, but instead the whole final result 😅